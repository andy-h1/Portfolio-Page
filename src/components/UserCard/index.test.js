import React from 'react';
import { render } from '@testing-library/react';
import { UserCard } from '.';

const props = {
  age: 25,
  email: 'test@gmail.com',
  id: '1234testid',
  name: 'usernameTest'
};

describe('User Card component', () => {
  it('should render', () => {
    const { container } = render(<UserCard {...props} />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
