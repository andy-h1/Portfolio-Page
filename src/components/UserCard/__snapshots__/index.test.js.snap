// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User Card component should render 1`] = `
<div
  class="sc-bdnylx cgnZNF"
>
  <div>
    <table
      class="sc-eCApGN eHUFhS"
    >
      <tbody>
        <tr>
          <th
            class="sc-jSFkmK gvAWUR"
          >
            Username:
          </th>
          <td
            class="sc-gKAblj kJjQBq"
          >
            usernameTest
          </td>
        </tr>
        <tr>
          <th
            class="sc-jSFkmK gvAWUR"
          >
            Email:
          </th>
          <td
            class="sc-gKAblj kJjQBq"
          >
            test@gmail.com
          </td>
        </tr>
        <tr>
          <th
            class="sc-jSFkmK gvAWUR"
          >
            Age:
          </th>
          <td
            class="sc-gKAblj kJjQBq"
          >
            25
          </td>
        </tr>
      </tbody>
    </table>
    <span
      class="sc-dlnjPT cmEWJv"
    >
      <button
        class="sc-hKFyIo dyckvF"
        type="button"
      >
        Edit
      </button>
      <button
        class="sc-hKFyIo dyckvF"
        type="button"
      >
        Delete
      </button>
    </span>
  </div>
</div>
`;
